.vscode/settings.json

```diff
@@ -26,6 +26,10 @@
     }
   ],
   "npm.packageManager": "yarn",
+  "editor.codeActionsOnSave": {
+    "source.organizeImports": false,
```

Comment:
we have a prettier plugin that does this for us, so organizeImports is no longer necessary